.env.example

```diff
@@ -2,7 +2,6 @@
 
 BITDRIFT_API_KEY=
 SENTRY_AUTH_TOKEN=
-EXPO_PUBLIC_ENV=development
```

Comment:
This will no longer be required to set. We can still set it for testflight and prod, otherwise it's assumed to be dev.